mods/d2k/rules/structures.yaml

```diff
@@ -599,7 +606,7 @@ starport:
 	Valued:
 		Cost: 1500
 	Building:
-		Footprint: xxx x=x =x=
+		Footprint: xxx xxx =x=
```

Comment:
Oh, I see. In that case I'll drop that commit, I just didn't expect this '=' to have a purpose.